src/main/scala/scorex/core/consensus/ModifierSemanticValidity.scala

```diff
@@ -10,20 +10,21 @@ object ModifierSemanticValidity {
     else if (code == Unknown.code) Unknown
     else if (code == Invalid.code) Invalid
     else Absent
-}
 
-case object Absent extends ModifierSemanticValidity {
-  override val code = 0
-}
+  case object Absent extends ModifierSemanticValidity {
+    override val code = 0.toByte
```

Comment:
It may be better to explicitly state the type of `code` and avoid the `toByte` method.
```
override val code: Byte = 0
```
same for similar definitions below.

Reply:
This works, thank you! I didn't notice that we are missing type annotation here.